website/content/en/docs/upgrading/v1beta1-migration.md

```diff
@@ -142,12 +142,12 @@ The [`karpenter-convert`](https://github.com/aws/karpenter/tree/release-v0.32.x/
 
 12. Roll over nodes: With the new NodePool yaml in hand, there are several ways you can begin to roll over your nodes to use the new NodePool:
 
-    1. Periodic Rolling with [Drift]({{< relref "../concepts/disruption#drift" >}}): Enable [drift]({{< relref "../reference/settings/#feature-gates" >}}) in Karpenter, then do the following:
+    1. Periodic Rolling with [Expiration]({{< relref "../concepts/disruption#automated-methods" >}}):
```

Comment:
Would it be possible to link to the Provisioner docs from a previous version here rather than the current NodePool docs? I believe the answer is no, but the reason I ask is it may be confusing to say later to set `ttlSecondsUntilExpired` and then link to the NodePool docs which have a different set of fields for disruption.

Reply:
Adding a note to make sure there's less confusion about how to do this.